server/http.go

```diff
@@ -51,6 +51,7 @@ type jsonResult struct {
 	ItemType       string
 	HumanIDs       []string
 	Row            string
+	RowKey         string
```

Comment:
Add comments (single line, on the right), that illustrate the difference between Row and RowKey. It looks to me like RowKey should be called RowField rather. Or perhaps, Row becomes RowKey, and RowKey becomes RowKeyField.
While we're at it, if you can think of useful comments for some of the other less obvious fields, such as HumanIDs, then add comments for them too.